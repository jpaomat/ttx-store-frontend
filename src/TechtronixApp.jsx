import './TechtonixApp.scss';
import { AppRouter } from './app/config/router/AppRouter';

export const 
TechtronixApp = ()=> {
  return (
    <>
      <AppRouter/>
    </>
  );
}
