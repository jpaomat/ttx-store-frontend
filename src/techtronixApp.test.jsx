import { TechtronixApp } from './TechtronixApp';
const Renderer = require('react-test-renderer/shallow');

const renderer = new Renderer();

describe('Techtronix component', () => {
	it('should render component', () => {
		const component = renderer.render(<TechtronixApp />);
		expect(component).toMatchSnapshot();
	});
});
