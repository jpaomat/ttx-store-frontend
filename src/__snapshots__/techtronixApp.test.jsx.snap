// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Techtronix component should render component 1`] = `
<React.Fragment>
  <AppRouter />
</React.Fragment>
`;
